test/bdd/pkg/messaging/messaging_sdk_steps.go

```diff
@@ -90,16 +92,16 @@ func (d *SDKSteps) sendMessage(fromAgentID, msg, msgType, purpose, toAgentID str
 		}
 	}
 
-	message := &genericInviteMsg{
+	msgMap := service.NewDIDCommMsgMap(&genericInviteMsg{
 		ID:      uuid.New().String(),
```

Comment:
I thought details like `ID` shouldn't need to be handled by higher layer code.

Reply:
Sure, the messenger will add it automatically in case if it absent. But, it will be done in my next PR.